package/crds/identity.aws.crossplane.io_iampolicies.yaml

```diff
@@ -78,6 +78,22 @@ spec:
                   path:
                     description: The path to the policy.
                     type: string
+                  tags:
+                    items:
+                      description: Tag represents a tag attached to a v1alpha1.User
```

Comment:
`description: Tag represents a tag attached to a v1alpha1.User`. Something went wrong here due to sharing the struct. Maybe the Tag struct doc strings could be changed to be generic?

Reply:
Sure @chlunde , I will make a change. I copied that bit from IAMRole crd.

Reply:
@chlunde  My bad, `make` is dynamically generating this description. And is consistent with `IAMRole` crd. Reverting to what was already there, as `make check-diff` is breaking.